code/modules/projectiles/projectile/bullets.dm

```diff
@@ -62,13 +62,15 @@
 	fire_sound = 'sound/weapons/Gunshot_c20.ogg'
 
 /obj/item/projectile/bullet/midbullet/lawgiver
-	damage = 10
+	damage = 15
 	stun = 0
 	weaken = 0
 	superspeed = 1
 
 /obj/item/projectile/bullet/midbullet2
-	damage = 25
+	damage = 20
+	stun = 0
+	weaken = 0
```

Comment:
I see that you added a new bullet, but you forgot to change the values on this one back.